specification/advisor/resource-manager/Microsoft.Advisor/stable/2023-01-01/advisor.json

```diff
@@ -248,6 +310,74 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Advisor/configurations/v2/{configurationName}": {
+      "put": {
+        "tags": [
+          "Configurations"
+        ],
+        "summary": "Create/Overwrite Azure Advisor configuration.",
+        "operationId": "Configurations_CreateInResourceGroupDuration",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "configContract",
+            "in": "body",
+            "description": "The Azure Advisor configuration data structure.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ConfigData"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroup",
```

Comment:
Consider instead referencing "ResourceGroupParameter" definition in common-types types.json Parameters section everywhere.